integration/kubernetes/multiMaster-embeddedJournal/job/alluxio-format-journal-job.yaml.template

```diff
@@ -30,7 +30,7 @@ spec:
     spec:
       containers:
         - name: alluxio-master
-          image: alluxio/alluxio:2.2.0-SNAPSHOT
+          image: alluxio/alluxio:2.3.0-SNAPSHOT
```

Comment:
This image version change was somehow overlooked in master branch.

Reply:
@ZacBlanco fyi